File: ios/wvwrapper/Models/WVViewModel.swift
//
//  WVViewModel.swift
//  wvwrapper
//
//  Created by SY L on 1/23/24.
//

import Foundation
class WVViewModel: NSObject, ObservableObject, WVDelegate {
    func greeting(param1: String) -> String {
        return "greeting + \(param1)"
    }
}

File: aos/app/src/main/java/xyz/lsy969999/wvwrapper/models/WVViewModel.kt
package xyz.lsy969999.wvwrapper.models

import androidx.lifecycle.ViewModel
import xyz.lsy969999.wvwrapper.views.WVI

class WVViewModel: ViewModel(), WVI {
    override fun greeting(param1: String): String {
        return "greeting + ${param1} z"
    }
}